packit_service/worker/tasks.py

```diff
@@ -333,6 +342,52 @@ def run_retrigger_bodhi_update(
     return get_handlers_task_results(handler.run_job(), event)
 
 
+@celery_app.task(
+    bind=True,
+    name=TaskName.vm_image_build,
+    base=HandlerTaskWithRetry,
+    queue="long-running",
+)
+def run_vm_image_build(self, event: dict, package_config: dict, job_config: dict):
+    handler = VMImageBuildHandler(
+        package_config=load_package_config(package_config),
+        job_config=load_job_config(job_config),
+        event=event,
+        celery_task=self,
+    )
+    return get_handlers_task_results(handler.run_job(), event)
+
+
+@celery_app.task(name=TaskName.vm_image_build_result, base=HandlerTaskWithRetry)
+def run_vm_image_build_result(
+    self, event: dict, package_config: dict, job_config: dict
+):
+    handler = VMImageBuildResultHandler(
+        package_config=load_package_config(package_config),
+        job_config=load_job_config(job_config),
+        event=event,
+    )
+    return get_handlers_task_results(handler.run_job(), event)
+
+
+@celery_app.task(
+    bind=True,
+    name="task.babysit_vm_image_build",
+    autoretry_for=(PackitVMImageBuildTimeoutException,),
+    retry_backoff=30,  # retry again in 30s, 60s, 120s, 240s...
+    retry_backoff_max=3600,  # at most, wait for an hour between retries
+    max_retries=14,  # retry 14 times; with the backoff values above this is ~8 hours
```

Comment:
I'm pretty sure we'll need to tune these in the future, but let's try it like this :)